### PR TITLE
Enable prepared statement product tests for Presto JDBC driver

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java [hunk 1]
```diff
@@ -91,334 +90,304 @@ public Requirement getRequirements(Configuration configuration)
...
-        }
+        assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + testValue)))
+                .containsOnly(row(testValue));
+        assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + "NULL")))
+                .hasNoRows();
+        assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + 2)))
+                .hasNoRows();
     }
 
     @Test(groups = {JDBC, SIMBA_JDBC})
```
Comment (kokosing) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java]: I think this tests no longer needs to be in both groups. I would remove `SIMBA_JDBC` and keep only `JDBC`.

Their use case is to run tempto with all JDBC tests but these that are SIMBA_JDBC (or PRESTO_JDBC depending on used driver). As I understand here we expect both drivers to be compatible.

Comment (findepi) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java]: @kokosing when testing simba, do we run sth like `-g JDBC -x PRESTO_JDBC` or `-g SIMBA_JDBC` ?

Comment (kokosing) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java]: `-g ldap,jdbc,simba_jdbc -x presto_jdbc,mysql,quarantine`

Comment (kokosing) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java]: or
`-g smoke,simba_jdbc,jdbc -x presto_jdbc,ldap,mysql,quarantine`

Generally rule is the same

**Pull request closed without merging**